apiserver/admin.go

```diff
@@ -227,11 +227,14 @@ func (a *admin) authenticate(req params.LoginRequest) (*authResult, error) {
 	if entity != nil {
 		if machine, ok := entity.(*state.Machine); ok && machine.IsManager() {
 			result.controllerMachineLogin = true
-			if machine.Tag() != a.srv.tag {
-				// We don't want to run pingers for other
-				// controller machines; they run their own.
-				startPinger = false
-			}
+			// TODO(axw) we shouldn't have to run pingers for
```

Comment:
For what its worth, with my new presence work we won't need to run these pingers for any connections.